src/dbus_mpris.rs

```diff
@@ -166,7 +166,11 @@ async fn create_dbus_server(
 
     // TODO: The first `true` allows us to replace orphaned dbus servers from previous sessions
     // later. We should instead properly release the name when the session ends.
```

Comment:
Could you move that comment below your insertion so that it stays next to the relevant line?